### DISCUSSION TITLE
A platform for backtesting, And in the future algo-trading.

### DISCUSSION THREAD

**Discussion by savakarrohan** [jupyterlite/jupyterlite#1300] [Show and tell]
Hey everyone, I was just hoping to show and tell my latest project. I call it StockWhiz. 

We at StockWhiz (https://www.stockwhiz.in), provide a back testing platform, with unlimited backtests, a ZeroCode interface, integrated with TradingView charts. 

At StockWhiz we help solve the first step of the algo-trading problem, that being backtesting. We aim to provide live algo-trading in the very near future. 
Here are some images showing the platform, this is potentially a version 1 of the product. 

ZeroCode interface:
![image](https://github.com/jupyterlite/jupyterlite/assets/50200523/d55f603c-a447-4b22-a7c1-3e2f9b79f218)

Integrated with TradingView Charts: 
![image](https://github.com/jupyterlite/jupyterlite/assets/50200523/52727701-4111-46a7-908a-97a4e20471ee)

A combination of both: 
![image](https://github.com/jupyterlite/jupyterlite/assets/50200523/676d7db3-0e16-43a3-a968-570c723ec3ae)

The story we are building towards: 
![image](https://github.com/jupyterlite/jupyterlite/assets/50200523/189662c8-df1a-47b1-b7e5-94ebac57e6c5)

I was out here to show the world our platform, and on the lookout if anyone is interested in contributing and being part of the founding team. With at least 60% of trades in the stock market being traded by computers and at least 90% of volumes generated on crypto exchanges as a result of algorithms. I want to bring algo-trading to the retail market. Would love to get on a call and potentially explain more to anyone interested. 

---
Drop a message here: savakarrohan@gmail.com

**Comment by jtpio** [comment]
Interesting, thanks @savakarrohan for sharing.

Curious: is this product using JupyterLite? The last screenshot might show a piece of it (and with blockly?), but not sure?

If this is indeed using JupyterLite underneath, maybe you can share a few more screenshots or screencasts to show it in action? That would help give ideas to other people of the kind of products that can be built with JupyterLite.
